app/src/main/java/com/sweng/scopehud/AboutActivity.java

```diff
@@ -0,0 +1,43 @@
+package com.sweng.scopehud;
+
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import android.widget.TextView;
+
+public class AboutActivity extends AppCompatActivity {
```

Comment:
This should extend the NavigationActivity rather than the AppCompatActivity,  and execute the necessary setup in onCreate() (just like MainActivity for example)